src/handlers/submitHandler.ts

```diff
@@ -0,0 +1,52 @@
+import { IHttp, IModify, IPersistence, IRead } from '@rocket.chat/apps-engine/definition/accessors';
+import { UIKitViewSubmitInteractionContext , UIKitInteractionContext} from '@rocket.chat/apps-engine/definition/uikit';
+import { ModalsEnum } from '../enums/Modals';
+import { MiroApp } from '../../MiroApp';
+import { SlashCommandContext } from '@rocket.chat/apps-engine/definition/slashcommands';
+import { getUIData } from '../lib/persistence';
+import { getBoards } from '../lib/get/getBoards';
+
+export class ExecuteViewSubmitHandler {
+	constructor(
+		private readonly app: MiroApp,
+		private readonly read: IRead,
+		private readonly http: IHttp,
+		private readonly modify: IModify,
+		private readonly persistence: IPersistence,
+        
+	) {}
+
+	public async run(
+        app:MiroApp,
+        context: UIKitViewSubmitInteractionContext,
+        read: IRead,
+        http: IHttp,
+        persistence: IPersistence,
+        modify: IModify,
+        slashcommandcontext?: SlashCommandContext, 
+        uikitcontext?: UIKitInteractionContext) {
+		const { user, view } = context.getInteractionData();
+        const uiData = await getUIData(read.getPersistenceReader(), user.id);
+        const room = uiData.room;
+        const roomId  =  uiData.room?.id
+        const data = context.getInteractionData();    
+        try {
+            
+		switch (view.id) {
```

Comment:
Careful with the indentation. Maybe we need to configure the prettier in the project.